components/Header.tsx

```diff
@@ -26,7 +27,11 @@ const Header = () => {
           </div>
         </a>
       </Link>
-      <div className="flex-none">
+      <motion.div
+        whileHover={{ scale: 1.1 }}
+        whileTap={{ scale: 0.9 }}
```

Comment:
I don't think this line is needed as there is a tap animation already on the button

Reply:
oka, I am removing that